plutus-contract/src/Plutus/Trace/Effects/RunContract.hs

```diff
@@ -87,7 +87,7 @@ type ContractConstraints s =
 walletInstanceTag :: Wallet -> ContractInstanceTag
 walletInstanceTag (Wallet i) = fromString $ "Contract instance for wallet " <> show i
 
--- | Run a Plutus contract (client side)
+-- | Run a Plutus contract (client side)activateContractWallet
```

Comment:
Looks like this accidentally slipped in.

Reply:
Thanks!